src/App.tsx

```diff
@@ -15,47 +16,108 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  SortFieldName = 'Name',
+  SortFieldLength = 'Length',
+}
+
+function getPrepearedGoods(goods: string[],
+  sortType: string, isReversed: boolean) {
```

Comment:
Type of the 'sortType' argument should correspond to the 'SortType' enum, not 'string'.